crates/re_viewer_context/src/space_view/scene_query.rs

```diff
@@ -7,6 +7,8 @@ pub struct SceneQuery<'s> {
     pub space_origin: &'s EntityPath,
 
     /// All queried entities.
+    ///
+    /// Contains also invisible objects, use `iter_entities` to iterate over visible ones.
     pub entity_paths: &'s IntSet<EntityPath>,
```

Comment:
did you check if any other scene part used this directly? should it be made private?

Reply:
considered and then moved on in the interest of time and breakage-risk.. but now you pushed me over it :P
Fixed. Doesn't look too risky, quickly testing all touched space view types